tools/eslint/.eslintrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  extends: ["plugin:@einride/default"],
+    rules: {
+    "jest/no-deprecated-functions": 0,
```

Comment:
Why is this needed? `jest` is not used.

Reply:
I know. The ESLint plugin however requires it: https://github.com/einride/eslint-plugin/blob/14dcb0b45a75a2dcf1d219aeafcb49f122c7af44/lib/index.js#L8

So I disabled it here. I think we should look into making our ESLint plugin a bit more modular with `peerDependencies`.

Reply:
So we would need to install `jest` so that the linter can run?

Reply:
Yeah...
Here's the output if you remove this disable: `Unable to detect Jest version - please ensure jest package is installed`.